tardis/gui.py

```diff
@@ -1,6 +1,39 @@
+"""Classes used to create the GUI for Tardis.
+
+This module contains all the classes and functions that create the GUI, but 
+it doesn't yet work outside IPython console. The console provides the event
+loop and the place to create/calculate the tardis model. So the module is
+basically a tool to visualize results. 
+
+Running instructions:
+--------------------
+    1. Decide which Qt binding you want to use (PySide or PyQt) and accordingly
+      set QT_API in shell::
+            export QT_API=pyside 
+            export QT_API=pyqt 
+    2. Start the IPython console with eventloop integration ::
+            ipython --pylab=qt4
+    3. Display your model::
+            from tardis import gui 
+            win = gui.Tardis()
+            win.show_model(mdl)
+
+Classes:
+-------
+    Tardis - The top level QMainWindow widget.
+
+Functions:
+---------
+    show_model(model) - A public method of Tardis(not gui itself) to take a 
+    model object and show its data.
```

Comment:
Oh I see! Thanks for being patient. I am converting these to numpy style
now. And I'll use sphinx to test if the doc builds. But I might still need
to bother you a few (and by a few I mean a lot :) more times.
On Jun 1, 2015 8:18 PM, "Kaushik Varanasi" <notifications@github.com> wrote:

> In tardis/gui.py
> <https://github.com/tardis-sn/tardis/pull/296#discussion_r31432286>:
>
> > +            export QT_API=pyqt
> > +    2. Start the IPython console with eventloop integration ::
> > +            ipython --pylab=qt4
> > +    3. Display your model::
> > +            from tardis import gui
> > +            win = gui.Tardis()
> > +            win.show_model(mdl)
> > +
> > +Classes:
> > +-------
> > +    Tardis - The top level QMainWindow widget.
> > +
> > +Functions:
> > +---------
> > +    show_model(model) - A public method of Tardis(not gui itself) to take a
> > +    model object and show its data.
>
> Attributes:
>   module_level_variable (int): Module level variables may be documented in
>     either the ``Attributes`` section of the module docstring, or in an
>     inline docstring immediately following the variable.
>
> see the indent after the modeule_level_variable line ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tardis-sn/tardis/pull/296/files#r31432286>.
>